### PR TITLE
Fixed the zombie apocalypse clause

### PR DESCRIPTION
**Pull request by JustusAdam**
The zombie apocalypse clause needed reworking.

I added the necessary changes.

Please merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,16 @@ Zum gemeinsamen Wohnen wird von den Unterzeichnern eine *Wohngemeinschaft* unter
...
+Die Wohngemeinschaft ist auf alle dem Zweck des Zusammenwohnens dienenden Tätigkeiten gerichtet.
+Sitz der Wohngemeinschaft ist Dresden.
 
 ### §2 Dauer der Wohngemeinschaft
 
-Die Wohngemeinschaft beginnt am 01.01.2016. Ihre Dauer ist unbestimmt. Die Mitbewohnervereinbarung kann unter Einhaltung einer Frist von 3,1 Monaten gekündigt werden. Die Kündigung muss mündlich erfolgen.
+Die Wohngemeinschaft beginnt am 01.01.2016.
+Ihre Dauer ist unbestimmt.
+Die Mitbewohnervereinbarung kann unter Einhaltung einer Frist von 3,1 Monaten gekündigt werden.
+Die Kündigung het mündlich zu erfolgen.
```
Comment (kiliankoe) [inline README.md]: "het"

File: README.md [hunk 2]
```diff
@@ -34,12 +39,17 @@ Im Falle des Auftretens einer übermächtigen künstlichen Intelligenz wird von
 
 ### §7 Zombieapokalypse
 
-(1) Für den Fall einer spontanen (oder nicht spontanen) Zombieapokalypse verpflichten sich die Mitbewohner die Wohnung zu verlassen und sich wahlweise im Keller oder oberen Stockwerken des Hauses zu verschanzen und gegen Untote zu verteidigen.
-(2) Sollte einer der Mitbewohner selbst von dem Virus oder der etwaigen anderen Ursache der Zombieapokalypse befallen sein ist dieser von den anderen Mitbewohnern zu isolieren aber nicht umzubringen.
+1. Für den Fall einer spontanen (oder nicht spontanen) Zombieapokalypse verpflichten sich die Mitbewohner die Wohnung zu verlassen und sich wahlweise im Keller oder oberen Stockwerken des Hauses zu verschanzen und gegen Untote zu verteidigen.
+2. Sollte einer der Mitbewohner selbst von dem Virus oder der etwaigen anderen Ursache der Zombieapokalypse befallen sein ist dieser von den anderen Mitbewohnern umgehend unschädlich zu machen.
+    Sofern möglich sollte dabei das Leben der infizierten Mitbewohner erhalten werden, für den Fall, dass ein Gegenmittel erfunden wird, jedoch nur, wenn dieß nicht Leib oder Leben der verbleibend menschlichen Mitbewohner.
```
Comment (kiliankoe) [inline README.md]: you accidentally a word?

File: README.md [hunk 3]
```diff
@@ -34,12 +39,17 @@ Im Falle des Auftretens einer übermächtigen künstlichen Intelligenz wird von
...
+
+    1. Sollte ein Person namens *Kilian Költzsch*, ungeachtet des Mitbewohnerstatus jener Person, mit dem Zombie Virus infiziert worden sein, so ist diese Person umgehend unschädlich zu machen.
+    Hierbei muss *nicht* beachtet werden ob zum fraglichen Zeitpunkt ein Gegenmittel oder die Möglichkeit der Erfindung eines Gegenmittels besteht.
```
Comment (kiliankoe) [inline README.md]: ಠ_ಠ

Comment (kiliankoe) [reply README.md]: "ein Person" btw, but this I'm not gonna merge. Feel free to keep this in your own fork :P

**Comment by JustusAdam** [comment]
Those are valid concerns, I hope I adequately addressed them. 🙂

**Comment by kiliankoe** [comment]
still got "ein Person" at the end there, but I'm fine with the comment :P

**Comment by kiliankoe** [comment]
lgtm

**Pull request merged**